procrastinate/contrib/aiopg/aiopg_connector.py

```diff
@@ -303,7 +303,11 @@ def _make_dynamic_query(self, query: str, **identifiers: str) -> Any:
 
     @wrap_exceptions()
     async def listen_notify(
-        self, on_notification: connector.Notify, channels: Iterable[str]
+        self,
+        on_notification: connector.Notify,
+        channels: Iterable[str],
+        *,
+        reconnect_interval: float = 2.0,
```

Comment:
It looks like it should be acceptably complex to implement the use of `reconnect_interval` in the AioPG Connector, do you want to do it ?